extensions/transmission/package.json

```diff
@@ -6,6 +6,11 @@
   "icon": "command-icon.png",
   "author": "FezVrasta",
   "license": "MIT",
+  "categories": [
+    "Applications",
+    "Media",
```

Comment:
I added media since one of the main uses of torrent is to, ahem, download Big Buck Bunny.